llvm/lib/SYCLLowerIR/ESIMD/LowerESIMD.cpp

```diff
@@ -505,6 +505,7 @@ class ESIMDIntrinDescTable {
         {"raw_send2_noresult",
          {"raw.send2.noresult",
           {a(0), a(1), ai1(2), a(3), a(4), a(5), a(6), a(7)}}},
+        {"wait", {"dummy.mov", {a(0)}}},
```

Comment:
I saw the llvm-test-suite test but should we also add a lit test for the lowering?

Reply:
The test https://github.com/intel/llvm-test-suite/pull/1615 does verify the lowering. 
Please see the "// CHECK: " lines there.

Reply:
Yeah there's some tradeoff because that's technically out of tree and doesn't get run with check-llvm, but I don't think it's a big deal